{
  "Header": {
    "Original": "CVE-2021-44420 (In Django 2.2 before 2.2.25, 3.1 before 3.1.14, and 3.2 before 3.2.10, ...)",
    "Line": 85525,
    "ID": "CVE-2021-44420",
    "Description": "(In Django 2.2 before 2.2.25, 3.1 before 3.1.14, and 3.2 before 3.2.10, ...)"
  },
  "Annotations": [
    {
      "Original": "- python-django 2:3.2.10-1",
      "Line": 85526,
      "Type": "package",
      "Package": "python-django",
      "Kind": "fixed",
      "Version": "2:3.2.10-1"
    },
    {
      "Original": "[bullseye] - python-django 2:2.2.25-1~deb11u1",
      "Line": 85527,
      "Type": "package",
      "Release": "bullseye",
      "Package": "python-django",
      "Kind": "fixed",
      "Version": "2:2.2.25-1~deb11u1"
    },
    {
      "Original": "[buster] - python-django \u003cnot-affected\u003e (Vulnerable code not present; is_endpoint support added later)",
      "Line": 85528,
      "Type": "package",
      "Release": "buster",
      "Package": "python-django",
      "Kind": "not-affected",
      "Description": "Vulnerable code not present; is_endpoint support added later"
    },
    {
      "Original": "[stretch] - python-django \u003cnot-affected\u003e (Vulnerable code not present; path converters added later)",
      "Line": 85529,
      "Type": "package",
      "Release": "stretch",
      "Package": "python-django",
      "Kind": "not-affected",
      "Description": "Vulnerable code not present; path converters added later"
    },
    {
      "Original": "NOTE: https://www.openwall.com/lists/oss-security/2021/12/07/1",
      "Line": 85530,
      "Type": "NOTE",
      "Description": "https://www.openwall.com/lists/oss-security/2021/12/07/1"
    },
    {
      "Original": "NOTE: https://www.djangoproject.com/weblog/2021/dec/07/security-releases/",
      "Line": 85531,
      "Type": "NOTE",
      "Description": "https://www.djangoproject.com/weblog/2021/dec/07/security-releases/"
    },
    {
      "Original": "NOTE: https://github.com/django/django/commit/333c65603032c377e682cdbd7388657a5463a05a (3.2.10)",
      "Line": 85532,
      "Type": "NOTE",
      "Description": "https://github.com/django/django/commit/333c65603032c377e682cdbd7388657a5463a05a (3.2.10)"
    },
    {
      "Original": "NOTE: https://github.com/django/django/commit/7cf7d74e8a754446eeb85cacf2fef1247e0cb6d7 (2.2.25)",
      "Line": 85533,
      "Type": "NOTE",
      "Description": "https://github.com/django/django/commit/7cf7d74e8a754446eeb85cacf2fef1247e0cb6d7 (2.2.25)"
    }
  ]
}